<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceClass</class>
 <widget class="QDialog" name="UserInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procedural Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/banners/gui/banner_form.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>65</number>
     </property>
     <property name="horizontalSpacing">
      <number>17</number>
     </property>
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Vertical Scaling (%)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="seedSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {
	width: 15px;
 	border: 0px;   
	border-radius: 2px;
 	margin: -1px 0;
 	background:#00a1ff;
	color: #0077bc;
}

QSlider::groove:horizontal {
	height: 20px; 
    border: 0px;
  	margin: 5px 0;
    background: qlineargradient(x1:0, y1:0, x2:1, y1:0, stop:0 #d4dadd, stop:1 #c1c8cc);
}	
</string>
       </property>
       <property name="maximum">
        <number>15000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Low End Computer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Persistence (%)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="octaveSpin">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    padding-right: 15px;
    border-width: 3;
	background: transparent;
	font-family: calibri;
	font-size: 15px;
	font-style: italic;
	
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
}

QSpinBox::down-button {
  	subcontrol-origin: border;
  	subcontrol-position: bottom right;
}

/* The following are left deliberately blank to disable them */

QSpinBox::up-button:hover {
  
}

QSpinBox::up-button:pressed {
 
}

QSpinBox::up-arrow {
 
}

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off { 
}



QSpinBox::down-button:hover {
 
}

QSpinBox::down-button:pressed {
   
}

QSpinBox::down-arrow {
  
}

QSpinBox::down-arrow:disabled, QSpinBox::down-arrow:off { 
}</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="lowBox">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    spacing: 5px;
	padding: 5px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="perSpin">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    padding-right: 15px;
    border-width: 3;
	background: transparent;
	font-family: calibri;
	font-size: 15px;
	font-style: italic;
	
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
}

QSpinBox::down-button {
  	subcontrol-origin: border;
  	subcontrol-position: bottom right;
}

/* The following are left deliberately blank to disable them */

QSpinBox::up-button:hover {
  
}

QSpinBox::up-button:pressed {
 
}

QSpinBox::up-arrow {
 
}

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off { 
}



QSpinBox::down-button:hover {
 
}

QSpinBox::down-button:pressed {
   
}

QSpinBox::down-arrow {
  
}

QSpinBox::down-arrow:disabled, QSpinBox::down-arrow:off { 
}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time of Day</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Seed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="seedSpin">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    padding-right: 15px;
    border-width: 3;
	background: transparent;
	font-family: calibri;
	font-size: 15px;
	font-style: italic;
	
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
}

QSpinBox::down-button {
  	subcontrol-origin: border;
  	subcontrol-position: bottom right;
}

/* The following are left deliberately blank to disable them */

QSpinBox::up-button:hover {
  
}

QSpinBox::up-button:pressed {
 
}

QSpinBox::up-arrow {
 
}

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off { 
}



QSpinBox::down-button:hover {
 
}

QSpinBox::down-button:pressed {
   
}

QSpinBox::down-arrow {
  
}

QSpinBox::down-arrow:disabled, QSpinBox::down-arrow:off { 
}</string>
       </property>
       <property name="maximum">
        <number>15000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Octaves</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="octaveSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {
	width: 15px;
 	border: 0px;   
	border-radius: 2px;
 	margin: -1px 0;
 	background:#00a1ff;
	color: #0077bc;
}

QSlider::groove:horizontal {
	height: 20px; 
    border: 0px;
  	margin: 5px 0;
    background: qlineargradient(x1:0, y1:0, x2:1, y1:0, stop:0 #d4dadd, stop:1 #c1c8cc);
}	
</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="scaleSpin">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
    padding-right: 15px;
    border-width: 3;
	background: transparent;
	font-family: calibri;
	font-size: 15px;
	font-style: italic;
	
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
}

QSpinBox::down-button {
  	subcontrol-origin: border;
  	subcontrol-position: bottom right;
}

/* The following are left deliberately blank to disable them */

QSpinBox::up-button:hover {
  
}

QSpinBox::up-button:pressed {
 
}

QSpinBox::up-arrow {
 
}

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off { 
}



QSpinBox::down-button:hover {
 
}

QSpinBox::down-button:pressed {
   
}

QSpinBox::down-arrow {
  
}

QSpinBox::down-arrow:disabled, QSpinBox::down-arrow:off { 
}</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="biomeMenu">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	height: 20px;

    border: 0px;
    border-radius: 2px;
    padding: 1px 15px 1px 5px;

	font-family: calibri;
	font-size: 15px;

	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #d4dadd, stop:1 #c1c8cc);
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 3px;
	border-radius: 2px;
}

QComboBox::drop-down {
    width: 0px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Grassland</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mountains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dunes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Glacier</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heatmap</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="perSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {
	width: 15px;
 	border: 0px;   
	border-radius: 2px;
 	margin: -1px 0;
 	background:#00a1ff;
	color: #0077bc;
}

QSlider::groove:horizontal {
	height: 20px; 
    border: 0px;
  	margin: 5px 0;
    background: qlineargradient(x1:0, y1:0, x2:1, y1:0, stop:0 #d4dadd, stop:1 #c1c8cc);
}	
</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="scaleSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {
	width: 15px;
 	border: 0px;   
	border-radius: 2px;
 	margin: -1px 0;
 	background:#00a1ff;
	color: #0077bc;
}

QSlider::groove:horizontal {
	height: 20px; 
    border: 0px;
  	margin: 5px 0;
    background: qlineargradient(x1:0, y1:0, x2:1, y1:0, stop:0 #d4dadd, stop:1 #c1c8cc);
}	
</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Biome</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="timeMenu">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	height: 20px;

    border: 0px;
    border-radius: 2px;
    padding: 1px 15px 1px 5px;

	font-family: calibri;
	font-size: 15px;

	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #d4dadd, stop:1 #c1c8cc);
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 3px;
	border-radius: 2px;
}

QComboBox::drop-down {
    width: 0px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Morning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Midday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Evening</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Midnight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00a1ff;

 	font: 14px;
	font-family: calibri;

    border-width: 0px;
    border-radius: 2px;
	padding: 3px;

    min-width: 5em;
}

QPushButton:pressed {
    background-color:#4193bf;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00a1ff;

 	font: 14px;
	font-family: calibri;

    border-width: 0px;
    border-radius: 2px;
	padding: 3px;

    min-width: 5em;
}

QPushButton:pressed {
    background-color:#4193bf;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00a1ff;

 	font: 14px;
	font-family: calibri;

    border-width: 0px;
    border-radius: 2px;
	padding: 3px;

    min-width: 5em;
}

QPushButton:pressed {
    background-color:#4193bf;
}</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="userinterface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>UserInterfaceClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UserInterfaceClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scaleSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scaleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>perSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>perSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>octaveSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>octaveSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>octaveSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>octaveSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>seedSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seedSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>seedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
